sparse/file.go

```diff
@@ -156,7 +156,7 @@ func alignmentShift(block []byte) int {
 	return int(uintptr(unsafe.Pointer(&block[0])) & uintptr(alignment-1))
 }
 
-func ReadDataInterval(file FileIoProcessor, dataInterval Interval) ([]byte, error) {
+func ReadDataInterval(file ReaderWriterAt, dataInterval Interval) ([]byte, error) {
```

Comment:
Rename the input to `rw` to make the naming consistent.

Reply:
Thanks Shuo! Fixed